src/components/views/rooms/AppsDrawer.tsx

```diff
@@ -279,7 +279,7 @@ export default class AppsDrawer extends React.Component<IProps, IState> {
             drawer = <PersistentVResizer
                 room={this.props.room}
                 minHeight={100}
-                maxHeight={(this.props.maxHeight || !widgetIsMaxmised) ? this.props.maxHeight - 50 : undefined}
+                maxHeight={this.props.maxHeight - 50}
```

Comment:
all changes to this file are unrelated. If you want me to I can make a different pr. It was just super convinient and simple to include them right after I found them without switching branches.